bom/test/pom.xml

```diff
@@ -69,8 +69,9 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-bom</artifactId>
-            <type>pom</type>
             <version>${project.version}</version>
+            <type>pom</type>
+            <scope>test</scope>
```

Comment:
@aloubyansky no sure why you didn't use scope test back then when introducing the bootstrap-bom?